tests/test_fuzz.py

```diff
@@ -91,25 +90,30 @@ def test_multicast_sender_with_random_data(self, data):
 			self.assertIsInstance(_fixture_port_num, int)
 			_fixture_SAY_args = [
 				"""SAY""",
-				"""--port""", str(_fixture_port_num),
-				"""--group""", """'224.0.0.1'""",
-				"""--message""", str("""'{d}'""").format(d=data)
+				"""--port""",
+				str(_fixture_port_num),
+				"""--group""",
+				"""'224.0.0.1'""",
+				"""--message""",
+				str("""'{d}'""").format(d=data)
```

Comment:
This could also be improved as per #134 with F-strings
```suggestion
				f"'{data}'"
```